src/Diagnostics.RuntimeHost/Services/SourceWatcher/Watchers/GitHubWatcher.cs

```diff
@@ -227,6 +230,8 @@ private async Task StartWatcherInternal(bool startup)
             }
             finally
             {
+                stopWatch.Stop();
+                Console.WriteLine("RunTime " + stopWatch.ElapsedMilliseconds);
```

Comment:
Remove this console logging. use LogMessage() method instead to log this to kusto.